system-test/dns.js

```diff
@@ -94,26 +92,26 @@ var DNS_DOMAIN = process.env.GCLOUD_TESTS_DNS_DOMAIN;
         'dns-admin.google.com.',
         '1 21600 3600 1209600 300'
       ].join(' ')
-    });
+    }),
 
-    records.spf = ZONE.record('spf', {
+    spf: ZONE.record('spf', {
       ttl: 21600,
       name: DNS_DOMAIN,
       data: 'v=spf1 mx:' + DNS_DOMAIN.replace(/.$/, '') + ' -all'
```

Comment:
Yeah it's because I moved the line that throws outside of a mocha hook.  Because it was in a hook of a skipped suite, it didn't run.